moveit_ros/benchmarks/src/BenchmarkOptions.cpp

```diff
@@ -275,18 +275,22 @@ void BenchmarkOptions::readPlannerConfigs(const rclcpp::Node::SharedPtr& node)
   for (const std::string& pipeline : pipelines)
   {
     std::string pipeline_name;
-    if (!node->get_parameter(np + "." + pipeline + ".name", pipeline_name))
+    std::string pipeline_parameter_name(np);
+    pipeline_parameter_name.append(".");
+    pipeline_parameter_name.append(pipeline);
+    pipeline_parameter_name.append(".name");
```

Comment:
Minor nit, this string could be a constant if you constructed it on one line.

```suggestion
    const std::string pipeline_parameter_name = np + '.' + pipeline + ".name";
```

Reply:
That's true, but then clang-tidy complains:
```
/root/target_ws/src/moveit2/moveit_ros/benchmarks/src/BenchmarkOptions.cpp:278:58: warning: string concatenation results in allocation of unnecessary temporary strings; consider using 'operator+=' or 'string::append()' instead [performance-inefficient-string-concatenation]
    const std::string pipeline_parameter_name = np + "." + pipeline + ".name";
                                                         ^
```

I could use a lambda to build the string the way I'm doing it here, and still have `pipeline_parameter_name` be const, but I don't know if that would give better performance than your solution (which seems to be clang-tidy's complaint).  What do you think?

Reply:
```suggestion
    constexpr auto pipeline_parameter_name = std::string(np).append('.').append(pipeline).append(".name");
```

Maybe that would quiet clang-tidy while still making the variable const without needing the lambda initialization pattern?

Reply:
I don't actually know if you can make that constexpr or if it just can be const, but if you can try making it constexpr.

Reply:
That worked (though I couldn't make it constexpr, only const).